test/unit/pipeline/number.ts

```diff
@@ -0,0 +1,26 @@
+/*
+	File: test/unit/pipeline/number.ts
+	cpuabuse.com
+*/
+
+/**
+ * Tests `number`.
+ */
+
+import { strictEqual } from "assert";
+import { processNumber } from "../../../src/pipeline/number";
+//@ts-ignore
+import { food as input } from "../../input/pipeline.js";
+//@ts-ignore
+import { food as expected } from "../../expected/pipeline.js";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected use of file extension "js" for "../../expected/pipeline.js"](https://app.codacy.com/gh/cpuabuse/cross-cat/pullRequest?prid=5042286)